src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -191,6 +197,19 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
     this._oauthService
       .updateOauthSession(value)
       .subscribe((requestInfoForm) => {
+        this._gtag
+          .reportEvent(
+            'RegGrowth',
+            'Sign-In',
+            'OAuth ' +
```

Comment:
so take a look at this part of the service function 

https://github.com/ORCID/orcid-angular/blob/5623901e4951f1a768277f2b1f3d5518c0a6da0a/src/app/core/google-analytics/google-analytics.service.ts#L64-L65

you can just send just the `requestInfoForm` as the third parameter, so the Oauth string gets build on the service.

Reply:
Ok